nodemon.json

```diff
@@ -0,0 +1,6 @@
+{
+    "watch": ["src"],
+    "ext": "ts",
+    "ignore": ["src/**/*.spec.ts"],
+    "exec": "ts-node ./src/index.ts"
+  }
```

Comment:
Extra ending blank line needed for all files

Reply:
Noted.